src/tallies/tally_scoring.cpp

```diff
@@ -95,15 +95,15 @@ FilterBinIter::operator++()
   // Find the next valid combination of filter bins.  To do this, we search
   // backwards through the filters until we find the first filter whose bins
   // can be incremented.
-  bool done_looping = true;
+  bool visited_all_combinations = true;
```

Comment:
This is a much better name!

Reply:
All thanks go to PullRequest Inc. for the suggestion!